Fix return_type parameter examples in ruby-client

meganemura opened the pull request:
### PR checklist

- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [ ] Ran the shell script under `./bin/` to update Petstore sample so that CIs can verify the change. (For instance, only need to run `./bin/{LANG}-petstore.sh` and `./bin/security/{LANG}-petstore.sh` if updating the {LANG} (e.g. php, ruby, python, etc) code generator or {LANG} client's mustache templates). Windows batch files can be found in `.\bin\windows\`.
- [ ] Filed the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master`, `3.4.x`, `4.0.x`. Default: `master`.
- [ ] Copied the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) to review the pull request if your PR is targeting a particular programming language.

### Description of the PR

Fix the wrong examples of return_type.
We need `<>` brackets for parsing return_type.
See: https://github.com/OpenAPITools/openapi-generator/blob/34945427d4179240816942fbfcac7b69042bee15/modules/openapi-generator/src/main/resources/ruby-client/api_client.mustache#L195-L204

wing328 commented:
@meganemura thanks for the PR. Can you please run `./bin/ruby-petstore.sh` or `bin\windows\ruby-petstore.bat` to update the Petstore samples so that CI can verify the change?

meganemura commented:
@wing328 

`./bin/ruby-petstore.sh` produces many diffs 😩 

In `bin/utils/ensure-up-to-date`, we runs `bin/openapi3/ruby-client-petstore.sh` to check whether the sample is up-to-date or not.
So I use `bin/openapi3/ruby-client-petstore.sh` to build sample client and commit it.

By the way, do we need `./bin/ruby-petstore.sh`? If not, I will remove it.

meganemura commented:
I think we need to update some java samples since the build of master branch is broken.

```
	modified:   samples/client/petstore/java/okhttp-gson-parcelableModel/src/main/java/org/openapitools/client/JSON.java
	modified:   samples/client/petstore/java/okhttp-gson/src/main/java/org/openapitools/client/JSON.java
	modified:   samples/client/petstore/java/rest-assured/src/main/java/org/openapitools/client/JSON.java
	modified:   samples/client/petstore/java/retrofit2/src/main/java/org/openapitools/client/JSON.java
	modified:   samples/client/petstore/java/retrofit2rx/src/main/java/org/openapitools/client/JSON.java
	modified:   samples/client/petstore/java/retrofit2rx2/src/main/java/org/openapitools/client/JSON.java

```

wing328 commented:
@meganemura thanks again for the PR, which has been included in the v3.3.3 release: https://twitter.com/oas_generator/status/1062929948191510528
